src/styles/blocks/nav.scss

```
@@ -0,0 +1,45 @@
+.nav {
+  &__list {
+    display: flex;
+    padding: 11px 0;
+  }
+
+  &__item {
+    list-style: none;
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    position: relative;
+    padding: 23px 0;
+
+    text-decoration: none;
+    line-height: 14px;
+    text-transform: uppercase;
+    color: #060b35;
+
+    &:hover {
+      color: #00acdc;
+      cursor: pointer;
+    }
+  }
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  display: block;
+  height: 4px;
+  top: 56px;
+  left: 0;
+  right: 0;
+
+  border-radius: 8px;
+  background: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
```

You could .is-active::after put in  .is-active
```suggestion
.is-active {
  color: #00acdc;
  
    &::after {
      content: "";
      position: absolute;
      display: block;
      height: 4px;
      top: 56px;
      left: 0;
      right: 0;
    
      border-radius: 8px;
      background: #00acdc;
   }
}


```

---

agree, use nesting